.travis.yml

```diff
@@ -24,6 +26,9 @@ notifications:
   email: false
 
 
+before_install:
+  - git submodule update --init -- es/upstream
```

Comment:
why do you have the second -- ?

Reply:
`man git-submodule` ? It's to specify the path to the submodule which should be initialized

Reply:
`git submodule update --init es/upstream` also works...